### PR TITLE
Created one companion object for configs

### PR DESCRIPTION
**Pull request by VladimirBaklan**
https://app.vivifyscrum.com/boards/61175/NEAF-42

### CODE REVIEW

File: src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala [hunk 1]
```diff
@@ -3,13 +3,32 @@ package com.itechart.ny_accidents.constants
 import com.typesafe.config.ConfigFactory
 
 object Configuration {
-  private val appConf = ConfigFactory.load("app.conf")
-  private val cacheConf = ConfigFactory.load("conf/cache.conf")
+  private lazy val appConf = ConfigFactory.load("conf/app.conf")
```
Comment (alexander-belyaev) [inline src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala]: Could you clarify why didn't you use default name application.conf  for main config file?
https://github.com/lightbend/config#standard-behavior

File: src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala [hunk 2]
```diff
@@ -3,13 +3,32 @@ package com.itechart.ny_accidents.constants
...
+  private lazy val cacheConf = ConfigFactory.load("conf/cache.conf")
+  private lazy val databaseConf = ConfigFactory.load("conf/ny_data_database.conf")
 
-  val CACHE_NAME: String = cacheConf.getString("cache.name")
-  val CACHE_PATH: String = "/home/" +
+  lazy val CACHE_NAME: String = cacheConf.getString("cache.name")
+  lazy val CACHE_PATH: String = "/home/" +
```
Comment (alexander-belyaev) [inline src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala]: To avoid string concatenation,  you could also use interpolation here

**Review by AlexKostritsa — APPROVED** [review]
good config

File: src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala (hunk 1 above)
Comment (VladimirBaklan) [reply src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala]: Fixed

File: src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala (hunk 2 above)
Comment (VladimirBaklan) [reply src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala]: Fixed

File: src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala (hunk 1 above)
Comment (alexander-belyaev) [reply src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala]: One more place object Configuration requires the same fix

Comment (VladimirBaklan) [reply src/main/scala/com/itechart/ny_accidents/constants/Configuration.scala]: +

**Review by alexander-belyaev — APPROVED** [review]

**Pull request merged**